Patch 1 (#2)
pcaversaccio/hardhat-project-template-ts: README.md

```bash
yarn install
```

## Running deployments 

```
npx hardhat xdeploy
```

> This starter use [xdeploy](https://github.com/pcaversaccio/xdeployer) check the documentation for more on deployments.

Reviewer: ```suggestion
This template uses the [xdeploy](https://github.com/pcaversaccio/xdeployer) Hardhat plugin. Check out the documentation for more information on the specifics of the deployments.
```

> ## env file 

Reviewer: ```suggestion
## `.env` File 
```

> In the .env file, place your wallets private key in the PRIVATE_KEY section, this allows access to your wallet to use both testnet and mainnet funds.

Reviewer: ```suggestion
In the `.env` file place the private key of your wallet in the `PRIVATE_KEY` section. This allows secure access to your wallet to use with both testnet and mainnet funds during Hardhat deployments. For more information on how this works, please read the documentation of the `npm` package [`dotenv`](https://www.npmjs.com/package/dotenv).
```

## Verification

> Change the contract address to your contract after deployment has been successful. This works for both testnet & mainnet. You will need to get an API key from etherscan.io, bscscan, snowtrace etc. 

Reviewer: ```suggestion
Change the contract address to your contract after the deployment has been successful. This works for both testnet and mainnet. You will need to get an API key from [etherscan](https://etherscan.io), [snowtrace](https://snowtrace.io) etc.
```

> Typescript deployment example:

Reviewer: ```suggestion
**Example:**
```

> npx hardhat verify --network fantomMain YOUR_CONTRACT_HERE --constructor-args deploy-arg.ts 

Reviewer: ```suggestion
```ts
npx hardhat verify --network fantomMain <YOUR_CONTRACT_ADDRESS> --constructor-args deploy-arg.ts
```
```

> Javascript deployment args example:

Reviewer: ```suggestion
```
